soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -1940,4 +1940,79 @@ enum chip_pll_mode {
 #define IT83XX_SPI_RX_VLISR          ECREG(IT83XX_SPI_BASE + 0x27)
 #define IT83XX_SPI_RVLI                    BIT(0)
 
+/**
+ *
+ * (20xxh) General Control (GCTRL) registers
+ *
+ */
+#ifndef __ASSEMBLER__
+struct gctrl_it8xxx2_regs {
+	/* 0x00-0x01: Reserved1 */
+	volatile uint8_t reserved1[2];
+	/* 0x02: Chip Version */
+	volatile uint8_t GCTRL_ECHIPVER;
```

Comment:
Please add some build asserts to validate the offsets are the expected value.  This ensures any future changes to the register map don't break the existing register locations.

Example here:
https://github.com/zephyrproject-rtos/zephyr/blob/5d988fde057f7d69d4e0ee658407326706f0f31b/soc/arm/nuvoton_npcx/common/reg/reg_def.h#L20
https://github.com/zephyrproject-rtos/zephyr/blob/5d988fde057f7d69d4e0ee658407326706f0f31b/soc/arm/nuvoton_npcx/common/registers.c#L12

Reply:
Done